dynpgext/test/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ add_postgresql_extension(
         VERSION 0.1
         SCRIPTS dynpgext_test--0.1.sql
         PRIVATE true
+        REQUIRES omni_ext
```

Comment:
Actually, I don't think it does. The [whole test](https://github.com/omnigres/omnigres/blob/master/dynpgext/test/tests/loader_present.yml) is just testing the lack of presence of the loader

Reply:
Updated.